src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -1664,468 +1661,1007 @@ static void vehicle_sprite_2_4(
-            break;
-        case 18:
-            vehicle_sprite_6_3(session, vehicle, imageDirection, z, vehicleEntry);
-            break;
-        case 19:
-            vehicle_sprite_6_4(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp42BankedRight90(session, vehicle, imageDirection, z, vehicleEntry);
             break;
+        default:
+            VehiclePitchUp42Unbanked(session, vehicle, imageDirection, z, vehicleEntry);
     }
 }
 
-#pragma endregion
-
-#pragma endregion
-
-#pragma region SteepSlopesDown
-
-// 6D4A05
-static void VehiclePitchDown42(
+// 6D4A31
+static void VehiclePitchUp60Unbanked(
     paint_session& session, const Vehicle* vehicle, int32_t imageDirection, int32_t z,
     const rct_ride_entry_vehicle* vehicleEntry)
 {
-    if (vehicleEntry->GroupEnabled(SpriteGroupType::Slopes42))
+    if (vehicleEntry->GroupEnabled(SpriteGroupType::Slopes60))
+    {
+        int32_t boundingBoxNum = (YawTo16(imageDirection)) + 40;
+        int32_t spriteNum = vehicleEntry->SpriteOffset(SpriteGroupType::Slopes60, imageDirection, 0);
+        VehicleSpritePaintWithSwinging(session, vehicle, spriteNum, boundingBoxNum, z, vehicleEntry);
+    }
+    else
+    {
+        VehiclePitchUp25(session, vehicle, imageDirection, z, vehicleEntry);
+    }
+}
+
+static void VehiclePitchUp60BankedLeft22(
+    paint_session& session, const Vehicle* vehicle, int32_t imageDirection, int32_t z,
+    const rct_ride_entry_vehicle* vehicleEntry)
+{
+    if (vehicleEntry->GroupEnabled(SpriteGroupType::Slopes60Banked22))
+    {
+        int32_t boundingBoxNum = (YawTo16(imageDirection)) + 40;
+        int32_t spriteNum = vehicleEntry->SpriteOffset(SpriteGroupType::Slopes60Banked22, imageDirection, 0);
+        VehicleSpritePaintWithSwinging(session, vehicle, spriteNum, boundingBoxNum, z, vehicleEntry);
+    }
+    else
+    {
+        VehiclePitchUp60Unbanked(session, vehicle, imageDirection, z, vehicleEntry);
+    }
+}
+
+static void VehiclePitchUp60BankedRight22(
+    paint_session& session, const Vehicle* vehicle, int32_t imageDirection, int32_t z,
+    const rct_ride_entry_vehicle* vehicleEntry)
+{
+    if (vehicleEntry->GroupEnabled(SpriteGroupType::Slopes60Banked22))
+    {
+        int32_t boundingBoxNum = (YawTo16(imageDirection)) + 40;
+        int32_t spriteNum = vehicleEntry->SpriteOffset(SpriteGroupType::Slopes60Banked22, imageDirection, 1);
+        VehicleSpritePaintWithSwinging(session, vehicle, spriteNum, boundingBoxNum, z, vehicleEntry);
+    }
+    else
+    {
+        VehiclePitchUp60Unbanked(session, vehicle, imageDirection, z, vehicleEntry);
+    }
+}
+
+static void VehiclePitchUp60(
+    paint_session& session, const Vehicle* vehicle, int32_t imageDirection, int32_t z,
+    const rct_ride_entry_vehicle* vehicleEntry)
+{
+    switch (vehicle->bank_rotation)
+    {
+        case 0:
+            VehiclePitchUp60Unbanked(session, vehicle, imageDirection, z, vehicleEntry);
```

Comment:
is it faster to have case 0 and default?

Reply:
I don't know, again this is just directly copied from the existing code. I don't think it would do any harm to remove that case if it is preferred.